nosnakecase.go

```diff
@@ -50,6 +50,12 @@ func report(pass *analysis.Pass, pos token.Pos, name string) {
 		return
 	}
 
+	// If prefix is Test or Benchmark, Fuzz, skip
+	// FYI https://go.dev/blog/examples
+	if strings.Contains(name, "Test") || strings.Contains(name, "Benchmark") || strings.Contains(name, "Fuzz") {
```

Comment:
I think this is too open, you will have false-negative now: if an Ident contains the word `Test` you will skip the check.
You should be more specific.

Reply:
at least a `HasPrefix` instead of `Contains`

Reply:
Oh, I've mistook. I'll fix it.